### PR TITLE
New package: arc-theme-20180114

### PR DESCRIPTION
**Pull request by jtalowell**

### CODE REVIEW

File: srcpkgs/arc-theme/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+pkgname=arc-theme
```
Comment (cr6git) [inline srcpkgs/arc-theme/template]: You forgot the first line: `# Template file for 'arc-theme'`

File: srcpkgs/arc-theme/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+version=20170302
+revision=1
+build_style=gnu-configure
+noarch=yes
+makedepends="pkg-config automake autoconf gtk+3-devel"
```
Comment (cr6git) [inline srcpkgs/arc-theme/template]:
````
hostmakedepends="automake pkg-config"
makedepends="gtk+3-devel"
depends="gnome-themes-standard gtk-engine-murrine"
````

File: srcpkgs/arc-theme/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+short_desc="Arc is a flat theme with transparent elements for GTK 3, GTK 2 and GNOME Shell"
```
Comment (cr6git) [inline srcpkgs/arc-theme/template]: `short_desc="Flat theme with transparent elements for GTK 3, GTK 2 and GNOME Shell"`

File: srcpkgs/arc-theme/template [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+maintainer="jtalowell <jtalowell@protonmail.com>"
+license="GPL-3.0"
```
Comment (cr6git) [inline srcpkgs/arc-theme/template]: `license="GPL-3.0-or-later"`

File: srcpkgs/arc-theme/template [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+homepage="https://github.com/horst3180/arc-theme"
+distfiles="https://github.com/horst3180/arc-theme/archive/${version}.tar.gz"
+checksum="9ca5c1d1ac267c4ae3725e84519d42bfb8c4c6934989839ebe3cb8699fbdfd45"
+
+pre_configure() {
+	NOCONFIGURE=1 ./autogen.sh
```
Comment (cr6git) [inline srcpkgs/arc-theme/template]:
````
build_style=gnu-configure
configure_script="./autogen.sh"
````

**Comment by jtalowell** [comment]
@cr6git Thank you for the corrections.

I've made the requested changes. Originally I didn't include `gnome-themes-standard` and `gtk-engine-murrine` as they are typically considered as optional dependencies.

I checked the manual but couldn't find anything detailing how to handle optional dependencies. Are they just listed under depends?

**Comment by Johnnynator** [comment]
xbps lacks the concept of optional dependencies. Sometimes a few are added to `depends` but mostly they are not added at all.

File: srcpkgs/arc-theme/template [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+build_style=gnu-configure
+configure_script="./autogen.sh"
+hostmakedepends="automake pkg-config"
+makedepends="gtk+3-devel"
+depends="gnome-themes-standard gtk-engine-murrine"
+short_desc="Flat theme with transparent elements for GTK 3, GTK 2 and GNOME Shell"
+maintainer="jtalowell <jtalowell@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/horst3180/arc-theme"
+distfiles="https://github.com/horst3180/arc-theme/archive/${version}.tar.gz"
```
Comment (travankor) [inline srcpkgs/arc-theme/template]: New upstream https://github.com/NicoHood/arc-theme

File: srcpkgs/arc-theme/template [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'arc-theme'
+pkgname=arc-theme
+version=20170302
+revision=1
+noarch=yes
+build_style=gnu-configure
+configure_script="./autogen.sh"
+hostmakedepends="automake pkg-config"
+makedepends="gtk+3-devel"
+depends="gnome-themes-standard gtk-engine-murrine"
```
Comment (travankor) [inline srcpkgs/arc-theme/template]: Optional dependencies are highly discouraged, unless you have a good reason

**Comment by jtalowell** [comment]
I've updated the package template with the new upstream and removed the optional dependencies.

**Comment by cr6git** [comment]
According to https://github.com/NicoHood/arc-theme/#manual-installation

"For the theme to function properly, install the following

    GNOME Shell 3.18 - 3.26, GTK 3.18 - 3.22
    The gnome-themes-standard package
    The murrine engine. This has different names depending on the distro."

**Comment by codingHahn** [comment]
@jtalowell please trigger a rebuild, because the RSA signature mismatch has been fixed. And add ```gnome-themes-standrad``` and ```gtk-engine-murrine``` to ```depends=```

File: srcpkgs/arc-theme/template [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'arc-theme'
+pkgname=arc-theme
+version=20180114
+revision=1
+noarch=yes
+build_style=gnu-configure
+configure_script="./autogen.sh"
+hostmakedepends="automake pkg-config"
+makedepends="gtk+3-devel sassc"
```
Comment (cr6git) [inline srcpkgs/arc-theme/template]: `sassc` ? what is it?

Comment (jtalowell) [reply srcpkgs/arc-theme/template]: It's a dependency required by the [new upstream](https://github.com/NicoHood/arc-theme/#manual-installation) - see [sassc](https://github.com/sass/sassc). The template for `sassc` doesn't restrict the architecture that it builds for so I'm unsure why it's unavailable for testing on `aarch64` and `armv7l`.

Comment (cr6git) [reply srcpkgs/arc-theme/template]: ok, I see...

https://travis-ci.org/void-linux/void-packages/jobs/398116820#L921

Try to put `sassc` in the `hostmakedepends` instead?

**Comment by cr6git** [comment]
@jtalowell it's good!

**Pull request merged**